config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
+
+  get '/rooms', to: 'rooms#index'
+
+  get '/guests/:id', to: 'guests#show'
+
+  post '/guest_rooms/:id', to: 'guest_rooms#create'
```

Comment:
Like I mentioned in the comment in the view, this is not a RESTful route. Remember that only show, delete, update have paths that end with an ID, create routes do not. The 2 options that you could have used here were:

```ruby
post "/guest_rooms", to: "guest_rooms#create"
```

or

```ruby
post "guests/:id/rooms", to: "guest_rooms#create"
```

The second one is more common, especially when we're talking about using the `resources` method and nesting resources. Something like this would produce the second route:

```ruby
resources :rooms, only: :index

resources :guests, only: :show do
  resources :rooms, only: :create, controller: "guest_rooms"
end
```